src/Collection/Collection.router.ts

```diff
@@ -400,27 +395,10 @@ export class CollectionRouter extends Router {
         // DCL Collections posts are being handled by the front-end at the moment and the backend updated using '/collections/:id/post'
         // TODO: Should this be halting the response? Retries?
 
-        if (dbCollection.forum_id) {
-          const postData = await getPost(dbCollection.forum_id)
-          await updatePost(
-            dbCollection.forum_id,
-            buildCollectionForumUpdateReply(
-              postData.raw,
-              result.items.slice(0, MAX_FORUM_ITEMS)
-            )
-          )
-        } else {
-          const { id: postId, link } = await createPost(
-            buildCollectionForumPost(
-              result.collection,
-              result.items.slice(0, MAX_FORUM_ITEMS)
-            )
-          )
-          await Collection.update<CollectionAttributes>(
-            { forum_link: link, forum_id: postId },
-            { id }
-          )
-        }
+        this.forumService.upsertThirdPartyCollectionForumPost(
```

Comment:
should we wait for this upsert?

Reply:
This was an error on my side. Thanks again!